packages/ingestion-validation/1.0.0/ingestion-validation-1.0.0.json

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
Should we commit this file - may be worth adding contents in the packages folder to .gitignore as we dont want to start storing LR artifacts here 

Also found out last week that for pipelines, the json only needs to include whats inside the config section otherwise it will not load properly when consumed from the hub

Reply:
good shout about adding it to .gitignore, will do